ask-sdk-dynamodb-persistence-adapter/lib/attributes/persistence/DynamoDbPersistenceAdapter.ts

```diff
@@ -73,34 +100,6 @@ export class DynamoDbPersistenceAdapter implements PersistenceAdapter {
         try {
             data = await this.dynamoDBDocumentClient.get(getParams).promise();
         } catch (err) {
-            if (err.code === 'ResourceNotFoundException' && this.createTable) {
```

Comment:
Maybe we can add similar error message as we do in java - get attributes method like [here](https://github.com/alexa/alexa-skills-kit-sdk-for-java/blob/2.0.x/ask-sdk-dynamodb-persistence-adapter/src/com/amazon/ask/attributes/persistence/impl/DynamoDbPersistenceAdapter.java#L83)

Reply:
I would prefer to throw the error at any conditions. In the Java Implementation, if the `AutoCreateTable` is set to true and developer deleted the table without re-instantiating the `PersistenceAdapter` the `getAttributes` will silently fail without showing any message

Reply:
Makes sense !! 👍